### ISSUE TITLE
oper mode and prefix

### ISSUE THREAD

**Issue by DarthGandalf** [kvirc/KVIrc#1112]
**Reported by Nakre on 19 Mar 2011 12:44:49 UTC**

```
(00:20:35) ircq-04.icq.com [*@*] has set mode: +Yyqo nick nick nick nick
(02:12:01) irc.icq.com [*@*] has set mode: +Y nick
(02:12:01) ChanServ [services@icq.com] has set mode: +qo nick nick
```

these are example of where the yY modes used
but at this one

```
(02:12:01) irc.icq.com [*@*] has set mode: mode +y < nick missing
```

when its used alone..
And then the y modes are going to "mode block" at top near topic block where it should not..

for the +y nick we need a ! as prefix that opers like chanserv or ircops etc.. to get on top of nick list..

---

Migrated from: https://svn.kvirc.de/kvirc/ticket/1112

**Comment by DarthGandalf** [comment]
**Comment by Nakre on 19 Mar 2011 13:08:25**
when someone gets the +Y mode we have the we have the mode Y masks button at top, i think thats useless and no needs just makes more buttons.. noone would use that window to -/+ a user prefix would you ? :x

**Comment by DarthGandalf** [comment]
**Comment by ctrlaltca on 20 Mar 2011 18:22:52**
fixed in r5657

---

Resolution: none → fixed
Status: new → closed

**Comment by DarthGandalf** [comment]
**Comment by Nakre on 05 Jul 2011 12:12:31**
i will keep this update as the issue sorted half, to keep ticket open.
because at this moment the +Y issue is sorted.
But the +y mode for the "!" prefix that opers get and where them to be above of channel owners as "!opernick".. on the nicklist is still need some work :)

---

Resolution: fixed → none
Status: closed → reopened

**Comment by DarthGandalf** [comment]
**Modified by ctrlaltca on 12 Jul 2011 16:01:47**
Version: none → 4.1 Equilibrium

**Comment by DarthGandalf** [comment]
**Modified by HelLViS69 on 12 Aug 2011 03:29:26**
Component: Core → IRC Protocol

**Comment by un1versal** [comment]
I suspect this may not be valid anymore @blindsighttf2

**Comment by staticfox** [comment]
I think this is still an inspircd thing. I'll look in to it in a bit.

**Comment by un1versal** [comment]
ping @staticfox and your milestone release claims also. https://github.com/kvirc/KVIrc/milestones/5.x.x%20-%20Release%20**BLOCKERS**

**Comment by TurkDesk** [comment]
Yes it's inspircd thing todo with https://wiki.inspircd.org/Modules/2.0/operprefix , still valid.

**Comment by wodim** [comment]
Anybody knows of a server that uses operprefix?

**Comment by TurkDesk** [comment]
librairc.net uses it.. bah my 2 years old ticket :)
